### PR TITLE
Update DR/GS string matching while guessing task types

### PR DESCRIPTION
**Pull request by khurtado**
This is related to the following WMCore issue: https://github.com/dmwm/WMCore/issues/10604

### CODE REVIEW

**Comment by khurtado** [comment]
@mrceyhun this small change should fix the UNKNOWN task types recently noticed, because they are missing 18DR/GS, 19DR/GS, etc. Could you please take a look?

File: src/htcondor_es/convert_to_json.py [hunk 1]
```diff
@@ -1001,45 +1001,49 @@ def recordTime(ad):
...
-    elif ttype.endswith("_1") or ttype.lower() == "reco":
-        return "RECO"
-    elif ttype == "MonteCarloFromGEN":
-        return "GENSIM"
-    elif ttype in ["DataProcessing", "Repack", "Express"]:
-        return ttype
+    jobType = ad.get("CMS_JobType", "UNKNOWN")
+
+    if jobType == "Processing":
+        return = "DataProcessing"
```
Comment (mrceyhun) [inline src/htcondor_es/convert_to_json.py]: No `=` after return

**Review by mrceyhun — COMMENTED** [review]
Thanks for the PR @khurtado . There is just a small syntax error, other than that ok for us. After the fix, @nikodemas can test and deploy to production.

File: src/htcondor_es/convert_to_json.py (hunk 1 above)
Comment (khurtado) [reply src/htcondor_es/convert_to_json.py]: @mrceyhun Fixed, thank you for noticing that!

**Comment by mrceyhun** [comment]
@nikodemas can you test this PR using Docker image and if everything is ok - you know what to look now, check condition results for data.TaskType - please merge this. And deploy it to production in vocms0240:
- Wait for both queue(*/12) and history (each 12 minutes with 5 mins delay) to finish in 12 minutes period.
- Check the doc: https://github.com/dmwm/cms-htcondor-es/blob/master/DEPLOYMENT.md#simple-deployment-procedure
- Don't forget to tail the logs for a while
- If something goes wrong, roll-back to previous commit: `git checkout "previous commit hash"`

**Pull request merged**